PaymentButtons/src/main/res/values/dimens.xml

```diff
@@ -24,6 +26,6 @@
     <dimen name="paypal_payment_button_label_text_size_large">18sp</dimen>
 
     <!-- Make logo height 76% of button -->
-    <item name="paypal_payment_button_logo_weight" type="dimen" format="float">0.76</item>
-    <item name="paypal_payment_button_vertical_spacer_weight" type="dimen" format="float">0.12</item>
```

Comment:
I thought the additional `type="dimen"` and `float="float"` attributes were necessary at one point, but I was able to verify using `<dimen />` by itself works fine.